src/model/article-history.v1.yaml

```diff
@@ -1,25 +1,50 @@
 $schema: http://json-schema.org/draft-04/schema#
-title: Article history
-type: object
-properties:
-    received:
-        type: string
-        format: date
-    accepted:
-        type: string
-        format: date
-    versions:
-        description: List of article versions, sorted by version in descending order
-        type: array
-        items:
-          - oneOf:
-              - $ref: ../snippets/article-poa.v1.yaml
-              - $ref: ../snippets/article-vor.v1.yaml
-        minItems: 1
-dependencies:
-    received:
-      - accepted
-    accepted:
-      - received
-required:
-    - versions
+oneOf:
+-
+    title: Article history
+    type: object
+    properties:
+        received:
+            type: string
+            format: date
+        accepted:
+            type: string
+            format: date
+        versions:
+            description: List of article versions, sorted by version in descending order
+            type: array
+            items:
+              - oneOf:
+                  - $ref: ../snippets/article-poa.v1.yaml
+                  - $ref: ../snippets/article-vor.v1.yaml
+            minItems: 1
+    dependencies:
+        received:
+          - accepted
+        accepted:
+          - received
+    required:
+        - versions
+-
+    title: Event
+    type: object
+    allOf:
+      - $ref: ../snippets/event.v1.yaml
```

Comment:
but this is a [real life event](https://github.com/elifesciences/api-raml/blob/develop/src/samples/event/v2/complete.json), not a state change

Reply:
a state change isn't a type of event? Are you objecting to my use of 'event' (which is fair enough, I didn't know that other event existed). I'm working with very little input here, if you have better ideas then please take a crack at it.

Reply:
I could call it HistoryEvent? or HistoricalEvent??

Reply:
I like `HistoryEvent`. You wouldn't need a `starts`, `ends` and `published` dates if you defined a new snippet, like the current `event` snippet requires.

Reply:
oh - I see what I've done. This is a typo, that `- $ref: ../snippets/event.v1.yaml` isn't meant to be there.

Reply:
actually it looks all kinds of messed up.

Reply:
ok, just pushed what I intended to commit. it doesn't `node compile` so I'm investigating why